src/common/ApiResponse/types.ts

```diff
@@ -1,8 +1,10 @@
 export type ApiEvent = {
+  id: string;
   isError: boolean;
   title: string;
   status: number;
   statusText: string;
+  level: "info" | "warn" | "error";
```

Comment:
The options for the level prop are inspired by the Logging levels.

I plan on extending the Notifications feature to allow displaying `Toast` messages in addition to `Notifications`. I'm thinking about adding an additional prop to specify if we should render the event as a Notification or Toast.